README.txt

```diff
@@ -22,3 +22,21 @@ Gallerya transforms the WordPress native post gallery into a full fledged slides
 
 * PHP 7.0 or later.
 
+== Contributors & Developers ==
+
+The following people have contributed to this plugin.
+
+* [Christoph Knopf]()
+* [Fabian Marz](https://profiles.wordpress.org/fabianmarz)
+* [Joan López](https://profiles.wordpress.org/juanlopez4691)
+* [Luca Pipolo](https://profiles.wordpress.org/lucapipolo)
+* [Sara Kozińska](https://profiles.wordpress.org/ixtrim)
+* [Daniel Kundwien](https://profiles.wordpress.org/tha_sun)
+* [Timo Gauweiler]()
+* [Tom Hare]()
+
+== Changelog ==
```

Comment:
1. Normally we would extract the latest changes as release notes for every new release (which needs a repo commit anyway, hence the wp-release script); see e.g. https://wordpress.org/plugins/mollom/#developers

1. In any case, when linking to the GitHub repo for the history, we should link to the releases (or at least tags) instead of the bare commit list.